snap/local/compile_native_wheels.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+# Cross-compile cryptography and cffi native wheels for arm64 and armhf architectures,
+# on the versions required by the current pinning of Certbot dependencies.
+# Wheels are stored in snap/local/packages folder to speed up cross-compilation of Certbot snap.
+set -ex
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+TARGET_ARCHS="arm64 armhf"
+
+rm -f "${DIR}/packages/*"
+
+# shellcheck source=common.sh
```

Comment:
You're welcome! It is enforced on `snapcore/snapcraft`, so I thought it would not harm to have it here also if we decide one day to add linting to bash scripts.